torch/fx/experimental/symbolic_shapes.py

```diff
@@ -5542,7 +5542,7 @@ def resimplify_floor_div(axioms: Dict[sympy.Expr, sympy.Expr]) -> None:
                 s in self.size_like,
                 self.oblivious_upper_bound_exclusive.get(s),
             )
-            for s in sorted(fs, key=lambda s: str(s))  # TODO: speed up sort?
+            for s in sorted(fs, key=str)  # TODO: speed up sort?
```

Comment:
remove the TODO?

Reply:
This is a micro-opt, while this improves it a bit, we may want to still revisit in the future, unless we are happy with the sorted performance now.

Reply:
Yeah, the comment about speeding up sort is more about avoiding stringifying all the symbols